res/TensorFlowLiteRecipes/NonMaxSuppressionV4_001/test.recipe

```diff
@@ -0,0 +1,53 @@
+operand {
+  name: "boxes"
+  type: FLOAT32
+  shape { dim: 10 dim: 4 }
+}
+operand {
+  name: "scores"
+  type: FLOAT32
+  shape { dim: 10 }
+}
+operand {
+  name: "max_output_size"
+  type: INT32
+  shape { }
+  filler { tag: "explicit" arg: "5" }
+}
+operand {
+  name: "iou_threshold"
+  type: FLOAT32
+  shape { }
+}
+operand {
+  name: "score_threshold"
+  type: FLOAT32
+  shape { }
+}
+operand {
+  name: "selected_indices"
+  type: INT32
+  shape { }
+}
+operand {
+  name: "valid_outputs"
+  type: INT32
+  shape { }
+}
+operation {
+  type: "NonMaxSuppressionV4"
+  input: "boxes"
+  input: "scores"
+  input: "max_output_size"
+  input: "iou_threshold"
+  input: "score_threshold"
+  output: "selected_indices"
+  output: "valid_outputs"
+}
+input: "boxes"
+input: "scores"
+input: "max_output_size"
```

Comment:
This operator has explicit filler, I am not sure this is correct

Reply:
![image](https://user-images.githubusercontent.com/3631351/89836535-a5ef5600-db6f-11ea-8b82-6a85e00aee8d.png)
ok, it works, but I think it will be better to exclude this input

Reply:
Didn't knew that providing a filler makes input redundant, nice to know :)